### PR TITLE
Add RNASeq de novo assembly

### PR DESCRIPTION
**Pull request by abretaud**
Some new material for RNASeq de novo assembly.
Currently there is only a dockerfile based on [this new docker flavor](https://github.com/abretaud/docker-galaxy-rnaseq-de-novo) (still building)

### CODE REVIEW

**Comment by bebatut** [comment]
Hi,

Thanks.

It would be better to add this material to the `RNA-Seq` directory.
Do you plan to add a tutorial (markdown and slides)?

Bérénice

**Comment by lecorguille** [comment]
@abretaud A thread about RNASeq denovo path: https://github.com/bgruening/training-material/issues/94

**Comment by abretaud** [comment]
Right, for now I only have the docker stuff, so do you think we should have a single docker for both ref-based and de novo rnaseq? Or keep the 2 separated (but how to store it in RNA-Seq?)
I don't have yet a tutorial, I can work on it, but later :smile:

**Comment by bebatut** [comment]
From my point of view, I would like to have only one docker. But maybe it is not the best solution. What do you think?

**Comment by abretaud** [comment]
Hehe, I hesitate too, that's why I ask :laughing: 
I'll see a bit later if I can switch to the single docker solution.

**Comment by bgruening** [comment]
A single container would lower the maintenance effort and would enable to make a comprehensive workshop about all RNAseq things with one instance.

The only reason I see against one single container is that it get to big at some point ... I'm not so worried about this currently :)

File: RNA-Seq-de-novo/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+This material is maintained by:
+
+- Anthony Bretaudeau (@abretaud)
+
+For any question related to this topic and the content, you can contact them.
+
+The following individuals have contributed to this training material:
+
+- Name 1
```
Comment (bgruening) [inline RNA-Seq-de-novo/README.md]: Your name :)

**Comment by abretaud** [comment]
Yep, and people will have some tools in the left that they won't use depending but it's probably not a problem.
And it's probably better to present the difference between ref-based and de novo in the slides/tutorial.
I'll rework this PR in a few days, I think I won't be able to do much more during this hackathon (sorry for that)

**Comment by bgruening** [comment]
@abretaud no problem. Thanks for your contribution!

File: RNA-Seq-de-novo/docker/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+[![Docker Stars](https://img.shields.io/docker/stars/bgruening/galaxy-training-exome-seq.svg?maxAge=2592000)](https://hub.docker.com/r/bgruening/galaxy-training-exome-seq/)
+
+Galaxy Docker repository for Exome sequencing training material :whale: :eyes:
+====
+
+:whale: This Docker Image contains everything what is needed to run this training, including an interactive Galaxy Tour.
+
+# Requirements
+
+- [Docker](https://docs.docker.com/installation/) for Linux / Windows / OSX
```
Comment (yhoogstrate) [inline RNA-Seq-de-novo/docker/README.md]: Are there any hardware requirements, e.g. RAM?

Comment (bgruening) [inline RNA-Seq-de-novo/docker/README.md]: Depending on the tools and the training data ... so hard to tell. In general no. Every special requirement should be stated in the readme file imho.

**Comment by abretaud** [comment]
Closing this PR, nothing valuable here, we'll rework this later

**Pull request closed without merging**